test.php

```diff
@@ -0,0 +1,97 @@
+<?php
+
+namespace src\Integration;
+
+class DataProvider
+{
+    private $host;
+    private $user;
+    private $password;
+
+    /**
+     * @param $host
+     * @param $user
+     * @param $password
+     */
+    public function __construct($host, $user, $password)
```

Comment:
missing type hinting. Should be `__construct(string $host, string $user, string $password)`